File: cs/sample/Sample/DuplicateEmails/DuplicateEmails.cs
using System;
using System.Collections.Generic;
using Microsoft.Data.Sqlite;

namespace Sample.DuplicateEmails{
    public class duplicateEmails{
        public static string naive(){
            string result = "";
            using var con = new SqliteConnection("Data Source=:memory:");
            con.Open();
            var cmd = con.CreateCommand();
            cmd.CommandText = @"CREATE TABLE Person(
                id INTEGER PRIMARY KEY,
                email TEXT)";
            cmd.ExecuteNonQuery();
            cmd.CommandText = "INSERT INTO Person VALUES (1,'a@b.com')";
            cmd.ExecuteNonQuery();
            cmd.CommandText = "INSERT INTO Person VALUES (2,'c@d.com')";
            cmd.ExecuteNonQuery();
            cmd.CommandText = "INSERT INTO Person VALUES (3,'a@b.com')";
            cmd.ExecuteNonQuery();

            cmd.CommandText = "SELECT email as Email FROM Person GROUP BY email HAVING COUNT(email)>1;";
            var rdr = cmd.ExecuteReader();

            while (rdr.Read())
            {
                result = rdr.GetString(0);
            }

            return result;
        }
    }
}

File: py/solns/duplicateEmails/duplicateEmails.py
import sqlite3 as sq3
class Solution:
    @staticmethod
    def naive():
        con = sq3.connect(':memory:')
        cur = con.cursor()
        cur.execute('''CREATE TABLE Person (id real,email text)''')
        cur.execute("INSERT INTO Person VALUES (1,'a@b.com')")
        cur.execute("INSERT INTO Person VALUES (2,'c@d.com')")
        cur.execute("INSERT INTO Person VALUES (3,'a@b.com')")
        con.commit()
        for row in cur.execute(
            'SELECT email as Email FROM Person GROUP BY email HAVING COUNT(email)>1;'):
            pass
        con.close()
        return row[0]